README.adoc

```diff
@@ -84,30 +84,37 @@ mobile-security-service-operator in your OpenShift cluster as follows:
 ....
 make cluster/prepare
 make install
+oc set env deployment/$(oc get -n mobile-developer-console -f deploy/operator.yaml --template "{{.metadata.name}}") OPENSHIFT_HOST="${OPENSHIFT_HOST}"
```

Comment:
I'm not sure if it's important here or not, but by the time this will cause the env var to be set in the operator, the operator may have already provisioned everything for the CR that was created from `make install` on the previous line.

Reply:
That's right...
We should actually do all of these in the `Makefile` so we can play with the order. But I will now merge without doing this. Not a blocker because RHMI installation will take care of these with Ansible.

Reply:
I tried to do some Makefile magic but failed miserably. Some bash magic would work I think, but honestly, separating `make install` into 2 tasks and keeping the instructions in README is much cleaner IMO.